app-ios/Native/Sources/Feature/Settings/SettingsScreen.swift

```diff
@@ -0,0 +1,189 @@
+import Component
+import Dependencies
+import Model
+import SwiftUI
+import Theme
+
+// TODO: Should use system font
```

Comment:
```suggestion
// NOTE: Should use system font
```

nits: Sorry if my review caused any misunderstanding. I think in this case using `NOTE` would be more natural than `TODO` to reflect the intent of the comment.

Reply:
I changed this comment as you said